test/Microsoft.AspNetCore.Server.Kestrel.Performance/configs/MsBuildExecutor.cs

```diff
@@ -0,0 +1,69 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Reflection;
+using BenchmarkDotNet.Characteristics;
+using BenchmarkDotNet.Diagnosers;
+using BenchmarkDotNet.Extensions;
+using BenchmarkDotNet.Jobs;
+using BenchmarkDotNet.Loggers;
+using BenchmarkDotNet.Running;
+using BenchmarkDotNet.Toolchains;
+using BenchmarkDotNet.Toolchains.DotNetCli;
+using BenchmarkDotNet.Toolchains.Results;
+
+namespace Microsoft.AspNetCore.Server.Kestrel.Performance
+{
+    public class MsBuildExecutor : IExecutor
```

Comment:
Awesome! I'll send you an email so we can arrange when and where to meet.

I wasn't able to run the generated binaries with what came out of the box as of your latest commit. I would get this error when trying to run the benchmarks, until it failed:

```
/Users/Cesar/.dotnet/sdk/1.0.0-rc3-004522/Microsoft.Common.CurrentVersion.targets(3935,5): warning MSB3026: Could not copy "obj/Release/netcoreapp1.1/Microsoft.AspNetCore.Server.Kestrel.Performance.dll" to "bin/Release/netcoreapp1.1/Microsoft.AspNetCore.Server.Kestrel.Performance.dll". Beginning retry 1 in 1000ms. The process cannot access the file '/Users/Cesar/src/aspnet/KestrelHttpServer/test/Microsoft.AspNetCore.Server.Kestrel.Performance/bin/Release/netcoreapp1.1/Microsoft.AspNetCore.Server.Kestrel.Performance.dll' because it is being used by another process. [/Users/Cesar/src/aspnet/KestrelHttpServer/test/Microsoft.AspNetCore.Server.Kestrel.Performance/Microsoft.AspNetCore.Server.Kestrel.Performance.csproj]
```

I also noticed that there's a mismatch between `DotNetCliBuilder` and `DotNetRunExecutor`: the builder puts binaries in `<generated project folder>\binaries`, while the executor calls `dotnet run`, which will re-compile the project and attempt to run them from `bin\Release\netcoreapp1.1`.